core/native/src/main/scala/zio/RuntimePlatformSpecific.scala

```diff
@@ -38,7 +39,10 @@ private[zio] trait RuntimePlatformSpecific {
   final val defaultReportFatal: Throwable => Nothing =
     (t: Throwable) => {
       t.printStackTrace()
-      throw t
+      try {
+        java.lang.System.exit(-1)
```

Comment:
I suspect this isn't correct. This would bring the whole process down. But that's not what it should do.
The ZIO runtime could be only a part of what runs in that given process.

Reply:
I just realised that it is done to test its behaviour anyhow will remove it